Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/RunScript.java

```diff
@@ -20,15 +22,30 @@ public RunScript(String scriptName) {
     			String cmdName = cmd[0];
     			String cmdArgs = cmd[1];
     			
+    			double[] point = new double[2];
+    			double rotation = 0;
+    			String parentheseless;
+    			String[] pointparts;
+    			if (AutoUtils.isDouble(cmdArgs)) {
+        			rotation = Double.valueOf(cmdArgs);
+        		} else if (AutoUtils.isPoint(cmdArgs)) {
+        			parentheseless = cmdArgs.substring(1, cmdArgs.length() - 1);
+        			pointparts = parentheseless.split(",");
+        			point[0] = Double.parseDouble(pointparts[0]);
+        		}
+    			
```

Comment:
This makes 3 different places in the code that we parse points. Can we centralize the parsing?

Reply:
I can consolidate two. I don't know if we should add parsing points to the other parsing, but I can ask Kevin.

Reply:
explain?

Reply:
Right now you just pass my code strings to represent points, which means I have to parse them a second time.